### DISCUSSION TITLE
Gitversion bases the version on the branch name

### DISCUSSION THREAD

**Discussion by rodrigoarnold** [GitTools/GitVersion#3839] [Q&A] [answered]
Hi everyone,

Gitversion is basing the version on the name of my release branch... For example: My release branch is called release/Sprint-2023-18-v1.7.7 and Gitversion is creating a version with the Major "2023", for because of the name of my release branch. **My question is, is there any way that Gitversion does not use my branch name?**

My GitVersion.yml:
 
assembly-versioning-scheme: MajorMinorPatchTag
assembly-file-versioning-scheme: MajorMinorPatchTag
mode: ContinuousDelivery
branches:
  develop:
    increment: Minor
  release:
    regex: ^releases?[/-]
    increment: Patch
ignore:
  sha: []
merge-message-formats: {}

I would be very grateful if someone could help me.

**Comment by HHobeck** [comment, marked as answer]
In the breaking changes notes you will find the following:

> A new configuration property with name version-in-branch-pattern has been introduced. This setting only applies on branches where the option is-release-branch is set to true. Please notice that the branch name needs to be defined after the version number by default (instead of support/lts-2.0.0 please name the branch like support/2.0.0-lts).

The default regular expression for version in branch pattern is `(?<version>[vV]?\d+(\.\d+)?(\.\d+)?).*`

This configuration has been introduces in release 6.0.0-beta.2:
- #2693

Happy Branching.

**Reply by rodrigoarnold** [reply]
Thank you very much, it really helped me a lot.
